src/external/adobe_coeff.c

```diff
@@ -622,6 +622,7 @@ static void dt_dcraw_adobe_coeff(const char *name, float cam_xyz[1][12])
     { "Samsung NX1", { 10686,-4042,-1052,-3595,13238,276,-464,1259,5931 } },
     { "Samsung WB2000", { 12093,-3557,-1155,-1000,9534,1733,-22,1787,4576 } },
     { "Samsung GX-1", { 10504,-2438,-1189,-8603,16207,2531,-1022,863,12242 } },
+    { "Samsung GX10", { 9566,-2863,-803,-7170,15172,2112,-818,803,9705 } }, /* copied from Pentax K10D */
```

Comment:
Please remove this, the camera only produces DNG's.

Reply:
In this case, I guess I should remove the line concerning the Samsung GX20?
It does only produces DNG as raw format too, but a line already exists in this file for it.

Reply:
Hmm. I think so, yes.

Reply:
ok, done!